MovieApp/MovieApp/App/DataRepository/Model/MovieDataRepositoryModel.swift

```diff
@@ -0,0 +1,14 @@
+struct MovieDataRepositoryModel {
```

Comment:
Naming goes like this:
UI -> <>ViewModel
UC -> <>Model
Repo -> <>RepoModel (or RepositoryModel)
DS -> <>DataModel
Client -> <>Request/Response

Reply:
I changed all the model names according to this template.
[MovieModel](https://github.com/lihter/Movie-App/pull/24/files#diff-b2ce96a4be225b4224eac0473fa85c4006296a2c071734276d6fa283d8777cb7R3)
[MovieRepoModel](https://github.com/lihter/Movie-App/pull/24/files#diff-16ec28a5469846b8ad26272a1925c9f70fa4bf8572c2a83c4b764c9caa874313R3)
[MovieDataModel](https://github.com/lihter/Movie-App/pull/24/files#diff-8619b2817b5d842b8b373e65db34f09b43f9568b4cbdc658cde0f60e682db4f0R1)
[MovieResponse](https://github.com/lihter/Movie-App/pull/24/files#diff-6a5a613e12bcfd2d3a5ab02c37bb5c630c30059bf525740739aeac557297c05fR11)